src/ServiceProvider.php

```diff
@@ -224,8 +223,14 @@ public function listenForJobProcessEvents()
 
     public function defineGateAbilities()
     {
-        Gate::define('graphql', function (Consumer $consumer, $operation) {
-            return $operation ? $consumer->tokenCan($operation) : false;
+        Gate::define('graphql', function ($user, $operation) {
+            $uses = class_uses_recursive($user);
+
+            if (in_array(\Laravel\Sanctum\HasApiTokens::class, $uses) && $operation) {
+                return $user->tokenCan($operation);
```

Comment:
Should we prefix these?

```suggestion
                return $user->tokenCan("graphql:{$operation}");
```

Reply:
No need.